ci/new_tsqa/tests/test_spdy_protocol_select.py

```diff
@@ -0,0 +1,141 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import requests
+import time
+import logging
+import subprocess
+
+import helpers
+
+import tsqa.test_cases
+import tsqa.utils
+import tsqa.endpoint
+
+log = logging.getLogger(__name__)
+
+class TestSPDY(helpers.EnvironmentCase):
+    environment_factory = {
+        'configure': {'enable-spdy': None},
+        'env': {'PKG_CONFIG_PATH': '/opt/spdylay/lib/pkgconfig/'},
+        }
+
+    @classmethod
+    def setUpEnv(cls, env):
+        '''
+        This function is responsible for setting up the environment for this fixture
+        This includes everything pre-daemon start
+        '''
+
+        # set up spdycat
+        build_dir = os.environ.get('top_builddir', '../..')
+        cls.client = '%s/spdylay/src/spdycat' % build_dir
+        log.info('top build_dir = {0}'.format(build_dir))
+        log.info('spdycat path = {0}'.format(cls.client))
+
+        # get spdy server ports
+        cls.spdy_port = tsqa.utils.bind_unused_port()[1]
+        log.info('spdy server port = {0}'.format(cls.spdy_port))
+        cls.http_port = tsqa.utils.bind_unused_port()[1]
+        log.info('http server port = {0}'.format(cls.http_port))
+
+        cls.configs['remap.config'].add_line('map / http://www.yahoo.com/\n')
```

Comment:
gonna comment on my own diff here to remind me to change this. (maybe have it point to the trafficserver docs?) what it points to is immaterial for these tests.